$OpenBSD: patch-notebook_base_zmqhandlers_py,v 1.1 2019/04/28 14:45:01 bket Exp $

Fix issue with py-tornado-5:

TypeError: __init__() got an unexpected keyword argument 'io_loop'

Index: notebook/base/zmqhandlers.py
--- notebook/base/zmqhandlers.py.orig
+++ notebook/base/zmqhandlers.py
@@ -173,7 +173,7 @@ class WebSocketMixin(object):
             self.last_ping = loop.time()  # Remember time of last ping
             self.last_pong = self.last_ping
             self.ping_callback = ioloop.PeriodicCallback(
-                self.send_ping, self.ping_interval, io_loop=loop,
+                self.send_ping, self.ping_interval,
             )
             self.ping_callback.start()
         return super(WebSocketMixin, self).open(*args, **kwargs)
